src/main/java/com/google/alpollo/EntityServlet.java

```diff
@@ -27,7 +27,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       String lyrics = request.getParameter(LYRICS);
 
       // Get the Entity list from the API
-      List<Entity> entityList = new ArrayList<>(AnalysisHelper.getEntityList(lyrics));
+      List<Entity> entityList = AnalysisHelper.getEntityList(lyrics);
```

Comment:
why you changed this and does it connected with this pr?

Reply:
All of these were changed in a previous PR, I don't know how they slipped past me. It's not necessary to make a copy of the list.

Reply:
You need to find a branch in which they're made and upload changes from it to this one

Reply:
They aren't changed on that branch either, maybe I just forgot or something. Isn't it better just to change it real fast from this branch?

Reply:
I think you can leave it as it is then